### DISCUSSION TITLE
how to  Enable dragging for the link

### DISCUSSION THREAD

**Discussion by reshin-pulapaddy** [clientIO/joint#2217] [Q&A]
### Introduction

how to  Enable dragging for the link?
is there any option to rotate the whole line by angle?
![link](https://github.com/clientIO/joint/assets/117301232/2af18805-7131-418c-b750-62883090ea8c)

I am using kitchen-sink-app and for link standard.Link
https://www.jointjs.com/demos/kitchen-sink-app

### Does your question relate to JointJS or JointJS+. Select both if applicable.

JointJS, JointJS+

**Comment by kumilingus** [comment]
> How to enable dragging for the link?

Use the interactivity option `linkMove: true` (See [docs](https://resources.jointjs.com/docs/jointjs/v3.7/joint.html#dia.Paper.prototype.options.interactive)).

> Is there any option to rotate the whole line by angle?

There is no built-in tool to rotate the links. You can try to implement it similarly to the rotate tool for elements:

https://codepen.io/jointjs/pen/eYLwGee
